geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-common/src/main/scala/org/locationtech/geomesa/fs/storage/common/StorageMetadata.scala

```diff
@@ -227,7 +227,16 @@ object StorageMetadata extends MethodProfiling with LazyLogging {
     * @param root path to the persisted metadata
     * @return
     */
-  def load(fc: FileContext, root: Path): Option[StorageMetadata] = Option(cache.get((fc, root)))
+  def load(fc: FileContext, root: Path , reload: Boolean = false): Option[StorageMetadata] = {
+    if(reload) {
+      val metadata = loadStorage(fc, root)
+      metadata match {
+        case Some(v) => cache.put((fc, root), v)
+        case None    => cache.put((fc, root), null)
+      }
+      metadata
+    }else Option(cache.get((fc, root)))
```

Comment:
could you clean up the formatting a bit here? there's an extra space before the comma on line 230, we try to put a space after an if, and we always try to use {} for an if/else. e.g.
```
if (reload) {
...
} else {
  ...
}
```